src/expression_evaluator/function_evaluator.cpp

```diff
@@ -21,8 +21,16 @@ void FunctionExpressionEvaluator::evaluate() {
     for (auto& child : children) {
         child->evaluate();
     }
+    auto expr = reinterpret_cast<binder::ScalarFunctionExpression*>(expression.get());
```

Comment:
You can pass the StringCastFunctionBindData directly to the function execution framework.
You don't need to extract the csvConfig there.